ci/lint.sh

```diff
@@ -42,6 +42,18 @@ if [ "$LINT" ]; then
     done
     echo "Linting *.pxi.in DONE"
 
+    echo "Linting *.pxd"
+    for path in 'src'
```

Comment:
this should recursively search _libs

Reply:
i think there may be some fixes needed as well to make this pass (to the actual .pxd files)

Reply:
@jreback Got it. Is there a way to make `lint.sh` run locally?

Reply:
not really but you can just check the command locally

Reply:
>  Is there a way to make lint.sh run locally?

On OSX/Linux:
```
export LINT=TRUE; /bin/bash ci/lint.sh
```

The first line of output is an error message you can ignore.